rust/cymbal/src/frames/mod.rs

```diff
@@ -15,17 +17,21 @@ pub mod resolver;
 // We consume a huge variety of differently shaped stack frames, which we have special-case
 // transformation for, to produce a single, unified representation of a frame.
 #[derive(Debug, Deserialize, Serialize, Clone)]
-#[serde(untagged)]
+#[serde(tag = "platform")]
 pub enum RawFrame {
+    #[serde(rename = "python")]
+    Python(RawPythonFrame),
+    #[serde(rename = "web:javascript")]
     JavaScript(RawJSFrame),
```

Comment:
Wonder if we should support `javascript` too so that we don't discount any earlier versions (at least for a couple of weeks)

Reply:
Done